### PR TITLE
feat(settings): tabs scrolls to top if navigated

### PR DESCRIPTION
**Pull request by dVp007**
solving! Scroll to top when switching settings tabs #681

### CODE REVIEW

**Comment by whyboris** [comment]
Thank you very much! The code looks like it will work 💯 -- I just want to test it out before merging 🤝

File: src/app/components/settings/settings.component.ts [hunk 1]
```diff
@@ -46,12 +46,14 @@ export class SettingsComponent implements OnInit {
...
     private translate: TranslateService
-  ) {}
+  ) { }
 
   ngOnInit(): void {
     this.additionalInput = this.appState.addtionalExtensions;
   }
-
+  ngOnChanges(): void {
+    document.getElementById('settings').scrollTop = 0;
```
Comment (whyboris) [inline src/app/components/settings/settings.component.ts:55]: I added `console.log()` above this and can confirm that this method gets called every time you switch between the _Tabs_ in _Settings_, but it doesn't seem to actually scroll to the top 😕 

ps, when you add `ngOnChanges()` method to an Angular class, you should also add it to the class declaration:
```ts
export class SettingsComponent implements OnInit, OnChanges {
```

**Review by whyboris — CHANGES_REQUESTED** [review]
Unfortunately with your changes, the app does not scroll to the top of the tab when switching tabs.

I wonder if this will be useful: https://stackoverflow.com/a/46348211/5017391
`.scrollIntoView()`

I tried using it with your code but it still didn't work. I'm unsure what is going wrong. Please let me know if you'd like help with this 🤝

**Comment by dVp007** [comment]
Okay will check it before can you send me a video or a GIF so that I can figure out what is wrong. Because when I tried it was scrolled at the top.

**Comment by whyboris** [comment]
To replicate:

scroll down to bottom, now click on the 1st tab:

notice the scrollbar is not at the top.
On your branch the scrollbar seems to move up a bit, but not to the top of the tab

**Comment by dVp007** [comment]
Can you please check i actually forgot to commit changes of a file. I guess it should work now.

**Pull request closed without merging**